static/robots.txt

```diff
@@ -0,0 +1,10 @@
+User-agent: *
+Disallow: /api/
+Disallow: /applications/
+Disallow: /apply/
```

Comment:
Is this definitely right? Might be nice for people to be able to find us via IG-related keywords?

Reply:
I am making a ticket to remove these once we have completed our voyage. I just wanted to block them from crawlers before they are "ready".